Dockerfile

```diff
@@ -67,3 +67,9 @@ RUN mkdir /opt/rwd/ && \
     | tar -xzC /opt/rwd --strip-components=1 && \
     cd /opt/rwd && \
     pip install --no-cache-dir -r requirements.txt
+
+RUN pip install flask==0.10.1
```

Comment:
I think that the requirement should go in the actual RWD repository since that's where the code which imports `Flask` lives.  It does seem like a nice option to not need to include the dependency if you're just running the script version, so +1 to the idea of a separate requirements file in that repo, which is then installed in the dockerfile here.